arangod/VocBase/LogicalCollection.cpp

```diff
@@ -1304,7 +1304,7 @@ auto LogicalCollection::getDocumentStateLeader() -> std::shared_ptr<
                      "Replicated state {} is not available as leader, accessed "
                      "from {}/{}. Status is {}.",
                      shardIdToStateId(name()), vocbase().name(), name(),
-                     to_string(leaderStatus->managerState.state));
+                     /* to_string(leaderStatus->managerState.state) */ "n/a");
```

Comment:
The manager state in its previous form does not quite fit to the new state managers. However, I'd like to implement some pendant, because I assume it's useful for observability.